docs/UserGuide.md

```diff
@@ -377,8 +386,7 @@ Action | Format, Examples
 **List all** | `list all`
 **List property** | `list property`
 **List appointment** | `list appointment`
-**Add new status** | `update INDEX new AMOUNT`<br><br> e.g.,`update 1 new 600000`
-**Update status** | `update INDEX u/STATUS`<br><br> e.g. `update 3 u/proceed`
+**Update status** | `update INDEX OPTION`<br><br> Options: <br><ul> <li>`u/new AMOUNT`</li> <li>`u/proceed`</li> <li>`u/cancel`</li> </ul> e.g.,`update 1 u/new 600000`
```

Comment:
There will be error parsing the html tags here. Try following what i did for the `find property` command below if you can. If not I will fix it after this is merged ba

Reply:
i tried my best to follow it